spec/factories/other_advice_method.rb

```diff
@@ -1,5 +1,6 @@
 FactoryGirl.define do
   factory :other_advice_method do
     name { Faker::Lorem.sentence }
+    sequence(:cy_name) { |n| "Dull gyngor arall #{n}" }
```

Comment:
Realistically reference data entries should be unique, so this ensures they are - same as you did for the accreditation and professional body models etc.